packages/accounts-base/accounts-base.d.ts

```diff
@@ -10,10 +10,10 @@ export interface URLS {
 }
 
 export interface EmailFields {
-  from?: ((user: Meteor.User) => string) | undefined;
-  subject?: ((user: Meteor.User) => string) | undefined;
-  text?: ((user: Meteor.User, url: string) => string) | undefined;
-  html?: ((user: Meteor.User, url: string) => string) | undefined;
+  from?: ((user: Meteor.User) => Promise<string>) | undefined;
```

Comment:
could you return a Promise<string> | string in that function definition, please?